packages/react-next/src/components/Link/Link.doc.tsx

```diff
@@ -45,6 +47,41 @@ export const LinkPageProps: IDocPageProps = {
         };
       },
     },
+    {
+      title: 'Link Themed',
+      code: LinkThemedExampleCode,
+      view: <LinkThemedExample />,
+      styles: ({ theme }) => {
```

Comment:
i don't think we want this example to be on doc site

Reply:
I need to put it in here to be able to see the changes when doing `yarn start`. Since the examples are easy to take out without much trouble I'd suggest keeping it here for testing purposes for the time being.

Reply:
storybook read `*.example.tsx`:
````
const req = require.context('../src/components', true, /\.Example\.tsx$/);
```

I think this `.doc.tsx` is for the website

Reply:
Removed.